src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -144,31 +144,31 @@ public void testFileExtensions() {
         final String[] fileExtensions = {"java"};
         final File pdfFile = new File("file.pdf");
         assertWithMessage("Invalid file extension")
-                .that(CommonUtil.matchesFileExtension(pdfFile, fileExtensions))
+                .that(CommonUtil.matchesFileExtension(pdfFile.toPath(), fileExtensions))
```

Comment:
instead of `pdfFile.toPath(),` can you create Path, instead of `new File("file.pdf");`